app/views/documents/show/_actions.html.erb

```diff
@@ -74,7 +76,9 @@
     <% end %>
 
     <% if @document.has_live_version_on_govuk? %>
-      <hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible" />
+      <% unless @document.user_facing_state == "removed" %>
```

Comment:
I'm not sure removing the line for just the removed state is going to work. Perhaps it would be better to add an action to create a new edition, as this seems to work for unpublished stuff.

Reply:
Updated and fixed-up